### PR TITLE
feat: wire-up confirmed "Add Liquidity" quote

### LINKED ISSUE #6019: Get quote for add / rem liquidity and add to ui component
#### AC 
1. quotes populated for adding and removing liquidity

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

Wires up the quote in confirm and status screens, bringing in the notion of "confirmed quote".

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

closes #6019

## Risk

None

## Testing

- Ensure the confirmed quote data is propagated over confirm and status screens

### Engineering

- ☝🏽 

### Operations

- ☝🏽 

#### RUNE asym

image
image
image
image

#### Asset asym

image
image
image

#### Symmetrical

image
image
image

### RUNE asym

image
image

### Asset asym

image
image
image

### Symmetrical

image
image
image

### CODE REVIEW

**Comment by gomesalexandre** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/shapeshift/web/6030?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* `develop`
  * **PR #6028** <a href="https://app.graphite.dev/github/pr/shapeshift/web/6028?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #6030** <a href="https://app.graphite.dev/github/pr/shapeshift/web/6030?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/6030?utm_source=stack-comment).

File: src/lib/utils/thorchain/lp/types.ts [hunk 1]
```diff
@@ -184,3 +184,13 @@ export type MidgardTvlHistoryResponse = {
   meta: MidgardTvlHistoryItem
   intervals: MidgardTvlHistoryItem[]
 }
+
+export type ConfirmedQuote = {
```
Comment (0xApotheosis) [inline src/lib/utils/thorchain/lp/types.ts:188]: We'll probably want to add gas and tx fees here, too.

**Review by 0xApotheosis — APPROVED** [review]
Does what is says on the box 👌

File: src/lib/utils/thorchain/lp/types.ts (hunk 1 above)
Comment (0xApotheosis) [reply src/lib/utils/thorchain/lp/types.ts:188]: I'll add it when I add fee logic 👌

**Pull request merged**